src/package/sign/sign.proj

```diff
@@ -91,9 +91,12 @@
       <AssembliesToSign Include="$(ArtifactsDirectory)Extensions\Microsoft.VisualStudio.TestTools.CppUnitTestFramework.CppUnitTestExtension.dll" />
 
       <!-- UWP runtime provider and dependent dlls -->
-      <AssembliesToSign Include="$(ArtifactsDirectory)CoreCon\VisualStudioDatastoreConfigurationProvider.dll" />
+      <!-- We removed this dll from package-->
+      <!--<AssembliesToSign Include="$(ArtifactsDirectory)CoreCon\VisualStudioDatastoreConfigurationProvider.dll" />-->
```

Comment:
If it's removed, please delete it, instead of commenting it.

Reply:
It is removed, but we will send the other team our nuget package to test without that dll. If it does not work will put this dll back. So will update comment itself.